src/views/download/download.jsx

```diff
@@ -0,0 +1,180 @@
+var React = require('react');
+var render = require('../../lib/render.jsx');
+
+var FormattedHTMLMessage = require('react-intl').FormattedHTMLMessage;
+var FormattedMessage = require('react-intl').FormattedMessage;
+
+var Page = require('../../components/page/www/page.jsx');
+var TitleBanner = require('../../components/title-banner/title-banner.jsx');
+var FlexRow = require('../../components/flex-row/flex-row.jsx');
+var SubNavigation = require('../../components/subnavigation/subnavigation.jsx');
+
+require('./download.scss');
+require('../../components/forms/button.scss');
+
+var Download = React.createClass({
+    type: 'Download',
+    render: function () {
+        return (
+            <div className="download">
+                <TitleBanner className="masthead">
+                    <div className="inner">
+                        <h1 className="title-banner-h1">
+                            <FormattedMessage id='download.title' />
+                        </h1>
+                        <p className="title-banner-p intro">
+                            <FormattedMessage id='download.intro' />
+                        </p>
+                        <p className="title-banner-p intro">
+                            <FormattedHTMLMessage id='download.introMac' />
+                        </p>
+                    </div>
+                    <div className="band">
+                        <SubNavigation className="inner">
+                            <a href="#installation">
+                                <li>
+                                    <FormattedMessage id='download.installation' />
+                                </li>
+                            </a>
+                            <a href="#updates">
+                                <li>
+                                    <FormattedMessage id='download.updatesTitle' />
+                                </li>
+                            </a>
+                            <a href="#other">
+                                <li>
+                                    <FormattedMessage id='download.otherVersionsTitle' />
+                                </li>
+                            </a>
+                            <a href="#issues">
+                                <li>
+                                    <FormattedMessage id='download.knownIssuesTitle' />
+                                </li>
+                            </a>
+                        </SubNavigation>
+                    </div>
+                </TitleBanner>
+                <div className="download-content">
+                    <div className="inner">
+                        <section id="installation">
+                            <span className="nav-spacer"></span>
+                            <FlexRow className="three-col-row">
```

Comment:
I think this needs to have a styling in the scss that says `align-items: flex-start`. Right now it inherits a `center` alignment, which leads to this:
<img width="930" alt="screen shot 2017-05-23 at 2 04 08 pm" src="https://cloud.githubusercontent.com/assets/1815772/26368821/c387ff3a-3fc0-11e7-9333-6be64b3b22eb.png">